### PR TITLE
Create Mediaeval Studies style

### PR DESCRIPTION
**Pull request by adunning**
Style for the journal [_Mediaeval Studies_](http://www.pims.ca/publications/journal-mediaeval-studies).

### CODE REVIEW

File: mediaeval-studies.csl [hunk 1]
```diff
@@ -0,0 +1,1347 @@
...
+    <id>http://www.zotero.org/styles/mediaeval-studies</id>
+    <link href="http://www.zotero.org/styles/mediaeval-studies" rel="self"/>
+    <link href="http://www.pims.ca/publications/journal-mediaeval-studies" rel="documentation"/>
+    <link href="https://www.zotero.org/styles/chicago-fullnote-bibliography" rel="template"/>
+    <author>
+      <name>Andrew Dunning</name>
+      <uri>https://orcid.org/0000-0003-0464-5036</uri>
+    </author>
+    <category citation-format="note"/>
+    <category field="generic-base"/>
```
Comment (adam3smith) [inline mediaeval-studies.csl]: switch out for a less generic field

File: mediaeval-studies.csl [hunk 2]
```diff
@@ -0,0 +1,1347 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" demote-non-dropping-particle="display-and-sort" page-range-format="chicago">
+  <info>
+    <title>Mediaeval Studies</title>
+    <id>http://www.zotero.org/styles/mediaeval-studies</id>
+    <link href="http://www.zotero.org/styles/mediaeval-studies" rel="self"/>
+    <link href="http://www.pims.ca/publications/journal-mediaeval-studies" rel="documentation"/>
+    <link href="https://www.zotero.org/styles/chicago-fullnote-bibliography" rel="template"/>
```
Comment (adam3smith) [inline mediaeval-studies.csl]: this should be http:// (no s) for legacy reasons

File: mediaeval-studies.csl [hunk 3]
```diff
@@ -0,0 +1,1347 @@
...
+    </author>
+    <category citation-format="note"/>
+    <category field="generic-base"/>
+    <issn>0076-5872</issn>
+    <eissn>2507-0436</eissn>
+    <summary>Mediaeval Studies journal</summary>
+    <updated>2018-02-25T19:59:17+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en-CA">
```
Comment (adam3smith) [inline mediaeval-studies.csl]: I'd probably to "en" here to catch all English locales

**Comment by adunning** [comment]
According to the website, it's identical to Chicago, but without publisher names or URLs. Should we not bother with a separate style?

**Comment by adam3smith** [comment]
no publisher names is a pretty significant change, we'll need a separate style.

**Comment by adam3smith** [comment]
@adunning it'd be great if you could make the couple of adjustments as per the above.

**Comment by adunning** [comment]
Have applied requested changes, if you can reassess.

**Comment by adunning** [comment]
It's quite confusing that the online CSL Validator doesn't pick up the same errors as Travis.

**Comment by adam3smith** [comment]
we run a set of additional tests beyond validation on styles for the repository -- naming, macros, etc. -- that aren't caught by just validation against the CSL schema.

**Comment by adunning** [comment]
I've gone through the process five times now, and it simply finds more errors each time that it didn't mention before. How does one deal with this normally?

**Comment by adam3smith** [comment]
I'm guessing that the macros you deleted contained the only call to other macros, which is why new issues came up. There's not a particularly easy way to handle this beyond checking this.

**Comment by adunning** [comment]
Obviously it likes you better.

**Comment by adam3smith** [comment]
Thanks!

**Pull request merged**